### ISSUE TITLE
Failed unmounting /var and /usr

### ISSUE THREAD

**Issue by niksajur** [systemd/systemd#5099]
### Submission type

  - [x] Bug report
  - [ ] Request for enhancement (RFE)

*NOTE: Do not submit anything other than bug reports or RFEs via the issue tracker!*

### systemd version the issue has been seen with

> ... 231

*NOTE: Do not submit bug reports about anything but the two most recently released systemd versions upstream!*

### Used distribution

> … Fedora 25 x86_64, kernel-4.9.3-200.fc25.x86_64

### In case of bug report: Expected behaviour you didn't see

> … Nice reboot and shutdown with all processes terminated and partitions unmounted.

### In case of bug report: Unexpected behaviour you saw

> … Failed unmounting /var and /usr at each and every one reboot or shutdown.
[Failed_unmounting.txt](https://github.com/systemd/systemd/files/712849/Failed_unmounting.txt)

### In case of bug report: Steps to reproduce the problem

> … Reboot or shutdown the system.

**Comment by ghost** [comment]
Hello,

I have the exact same error and been trying to find a solution for some time now.

There are some differences on my machine though:
- only /var partition shows errors in /var/log/messages after boot
- I run CentOS 7.3 (no updates, no customizations, just a fresh install)
- I run an older version of systemd version (comes installed by default on CentOS 7.3)


### systemd version the issue has been seen with

systemd-219-30.el7.x86_64



### Used distribution

CentOS Linux release 7.3.1611 (Core)
3.10.0-514.2.2.el7.x86_64

### In case of bug report: Expected behaviour you didn't see

> … Mount all of my partitions. Because of the error on /var partition at boot, all my "data" partitions do not get mounted (/home and /cache in my case)

### In case of bug report: Unexpected behaviour you saw

> … Failed unmounting cleanly /var, thus creating a cascade of problems with mount after reboot. Because it cannot 

### In case of bug report: Steps to reproduce the problem

> …
On my machine:
1. Install a fresh CentOS 7.3 (latest ISO)
2. After install, reboot
3. Check /var/log/messages


In my case, a strange thing is happening. If I try to set a more verbose logging from systemd, the mount errors go away and everything works fine. The steps to put systemd on debug mode I used are:

1. Uncomment the following options in /etc/systemd/journald.conf
```
RateLimitInterval=30s
RateLimitBurst=9000000
```

2. Restart journald service:
`systemctl restart systemd-journald`

3. Edit /etc/rsyslog.conf:
```
$ModLoad imjournal
$imjournalRatelimitInterval 0
$imjournalRatelimitBurst 0
```

4. Restart rsyslog:
`systemctl restart rsyslog`

5. Add boot kernel parameter for systemd in vim /boot/efi/EFI/centos/grub.cfg.
Append option "systemd.log_level=debug" to vmlinuz line


Does anyone have any idea why this thing is happening and how can I solve it permanently without leaving systemd on debugging mode?

**Comment by mbiebl** [comment]
Looks like a duplicate of #867

**Comment by ghost** [comment]
I've tried your solution but it doesn't fully work for me.

Added in /usr/lib/systemd/system/systemd-journald.service the line in bold:

_[Unit]
Description=Journal Service
Documentation=man:systemd-journald.service(8) man:journald.conf(5)
DefaultDependencies=no
Requires=systemd-journald.socket
After=systemd-journald.socket syslog.socket
Before=sysinit.target
**RequiresMountsFor=/var/log/journal**_

Right now, the errors that I see in messages are dbus related:
```
Jan 18 22:58:09 myserver NetworkManager[1578]: <warn>  [1484773089.4599] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.DBus: The connection is closed); only the private D-Bus socket will be available
Jan 18 22:58:12 myserver NetworkManager[1578]: <warn>  [1484773092.4605] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.DBus: The connection is closed); only the private D-Bus socket will be available
Jan 18 22:58:15 myserver NetworkManager[1578]: <warn>  [1484773095.4600] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.DBus: The connection is closed); only the private D-Bus socket will be available
Jan 18 22:58:18 myserver NetworkManager[1578]: <warn>  [1484773098.4606] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.DBus: The connection is closed); only the private D-Bus socket will be available
Jan 18 22:58:21 myserver NetworkManager[1578]: <warn>  [1484773101.4601] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.DBus: The connection is closed); only the private D-Bus socket will be available
```

Here are a few service statuses:

```
[root@myserver log]# systemctl status dbus.service
● dbus.service - D-Bus System Message Bus
   Loaded: loaded (/usr/lib/systemd/system/dbus.service; static; vendor preset: disabled)
   Active: inactive (dead) since Wed 2017-01-18 23:03:21 EET; 23s ago
  Process: 1579 ExecStart=/bin/dbus-daemon --system --address=systemd: --nofork --nopidfile --systemd-activation (code=exited, status=0/SUCCESS)
 Main PID: 1579 (code=exited, status=0/SUCCESS)


Jan 18 23:02:50 myserver.domain.tld systemd[1]: Starting D-Bus System Message Bus...
Jan 18 23:02:50 myserver.domain.tld dbus[1579]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service'
Jan 18 23:02:50 myserver.domain.tld dbus-daemon[1579]: dbus[1579]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service'
Jan 18 23:02:50 myserver.domain.tld dbus[1579]: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
Jan 18 23:02:50 myserver.domain.tld dbus-daemon[1579]: dbus[1579]: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
Jan 18 23:02:50 myserver.domain.tld dbus[1579]: [system] Activating via systemd: service name='org.freedesktop.hostname1' unit='dbus-org.freedesktop.hostname1.service'
Jan 18 23:02:50 myserver.domain.tld dbus-daemon[1579]: dbus[1579]: [system] Activating via systemd: service name='org.freedesktop.hostname1' unit='dbus-org.freedesktop.hostname1.service'
Jan 18 23:02:50 myserver.domain.tld dbus[1579]: [system] Successfully activated service 'org.freedesktop.hostname1'
Jan 18 23:02:50 myserver.domain.tld dbus-daemon[1579]: dbus[1579]: [system] Successfully activated service 'org.freedesktop.hostname1'
Jan 18 23:03:21 myserver.domain.tld systemd[1]: Stopping D-Bus System Message Bus...
```

```
[root@myserver log]# systemctl status dbus.socket
● dbus.socket - D-Bus System Message Bus Socket
   Loaded: loaded (/usr/lib/systemd/system/dbus.socket; static; vendor preset: disabled)
   Active: inactive (dead) since Wed 2017-01-18 23:03:21 EET; 1min 54s ago
   Listen: /var/run/dbus/system_bus_socket (Stream)

Jan 18 23:02:50 myserver.domain.tld systemd[1]: Listening on D-Bus System Message Bus Socket.
Jan 18 23:02:50 myserver.domain.tld systemd[1]: Starting D-Bus System Message Bus Socket.
```

```
[root@myserver log]# systemctl status dbus-org.freedesktop.NetworkManager
● NetworkManager.service - Network Manager
   Loaded: loaded (/usr/lib/systemd/system/NetworkManager.service; enabled; vendor preset: enabled)
   Active: active (running) since Wed 2017-01-18 23:02:50 EET; 2min 56s ago
     Docs: man:NetworkManager(8)
 Main PID: 1587 (NetworkManager)
   CGroup: /system.slice/NetworkManager.service
           └─1587 /usr/sbin/NetworkManager --no-daemon

Jan 18 23:05:18 myserver.domain.tld NetworkManager[1587]: <warn>  [1484773518.2350] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.... be available
Jan 18 23:05:21 myserver.domain.tld NetworkManager[1587]: <warn>  [1484773521.2347] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.... be available
Jan 18 23:05:24 myserver.domain.tld NetworkManager[1587]: <warn>  [1484773524.2343] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.... be available
Jan 18 23:05:27 myserver.domain.tld NetworkManager[1587]: <warn>  [1484773527.2350] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.... be available
Jan 18 23:05:30 myserver.domain.tld NetworkManager[1587]: <warn>  [1484773530.2348] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.... be available
Jan 18 23:05:33 myserver.domain.tld NetworkManager[1587]: <warn>  [1484773533.2344] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.... be available
Jan 18 23:05:36 myserver.domain.tld NetworkManager[1587]: <warn>  [1484773536.2349] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.... be available
Jan 18 23:05:39 myserver.domain.tld NetworkManager[1587]: <warn>  [1484773539.2347] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.... be available
Jan 18 23:05:42 myserver.domain.tld NetworkManager[1587]: <warn>  [1484773542.2344] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.... be available
Jan 18 23:05:45 myserver.domain.tld NetworkManager[1587]: <warn>  [1484773545.2341] bus-manager: could not create org.freedesktop.DBus proxy (Error calling StartServiceByName for org.freedesktop.... be available
Hint: Some lines were ellipsized, use -l to show in full.
```


```
[root@myserver log]# systemctl status tuned.service
● tuned.service - Dynamic System Tuning Daemon
   Loaded: loaded (/usr/lib/systemd/system/tuned.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Wed 2017-01-18 23:03:21 EET; 2min 57s ago
  Process: 1882 ExecStart=/usr/sbin/tuned -l -P (code=exited, status=1/FAILURE)
 Main PID: 1882 (code=exited, status=1/FAILURE)

Jan 18 23:03:00 myserver.domain.tld systemd[1]: Starting Dynamic System Tuning Daemon...
Jan 18 23:03:01 myserver.domain.tld systemd[1]: Started Dynamic System Tuning Daemon.
```


Let me know what other info must I provide (if necessary).

**Comment by niksajur** [comment]
Putting the line

`RequiresMountsFor=/var/log/journal`

to systemd-journald.service doesn't fix my problem. This "workaround" just hides the problem. The error messages are not logged in journal anymore and cannot be queried by journalctl. That's all, because systemd continues to fail unmounting /var and /usr and I can clearly see the error messages on the screen during reboot or shutdown. 

This is a bug and there is no sense to search for workarounds in order to simulate clean shutdown if it is not clean. After all it is systemd's basic responsibility to safely terminate all processes, securely unmount all partitions and cleanly shutdown the system.

**Comment by ghost** [comment]
Does my workaround work for you? I mean puting systemd + journald + rsyslog with the settings I provided and then reboot a couple of times.

**Comment by niksajur** [comment]
No, your workaround doesn't work for me. Furthermore, instead of two, now I have three error messages visible on the screen during reboot and logged in journal and /var/log/messages:

`systemd[1]: Failed unmounting RPC Pipe File System.`
`...`
`systemd[1]: Failed unmounting /var.`
`...`
`systemd[1]: Failed unmounting /usr.`

not to mention other error messages not related to this topic.

**Comment by ghost** [comment]
I see. Also, I forgot to mention that I don't have anything added in systemd-journald.service from mbiebl's solution, such as:

`RequiresMountsFor=/var/log/journal`

For me, it's a fresh install + systemd on debugging from the steps I liste above and the errors seem to go away (so far). I can confirm this on CentOS 7.3 and Cloudlinux 7.3. I've rebooted the server 5-10 times on each install to see if the errors appear again, but they didn't. I've also ran _yum update_ and rebooted, but still no errors.

Although this workaround works for me, it would be nice to actually have a real and concrete solution for this. I've tried a few solutions and suggestions from the archlinux forums, but none of them helped me.

**Comment by niksajur** [comment]
Yes, it would be nice they finally fix that awful, well known and more than two years old bug. Unfortunately I am afraid we are fighting a losing battle against lousy software and its insane code. It's not easy to debug a huge binary init system that tends to be an operating system, even more than operating system, with enormous amount of unnecessary complexity crammed into pid 1. IMHO, systemd is defective by design just like Gnome 3. Frankly speaking, systemd and Gnome 3 are two insane disasters for entire Linux community. They are killing Linux. Just look at Fedora 25. What is it? It is a nice Windows 10 Linux Edition missing just Registry (so far). If I have to choose between original and its imitation, I will choose the original. After all Microsoft Windows 10 is much better windows than Fedora 25.

**Comment by fsateler** [comment]
Maybe `/usr` and `/var` should get `DefaultDependencies=no` , and have `systemd-shutdown` unmount them (which is already happening).

**Comment by niksajur** [comment]
@fsateler Nice. What is the right place to put `DefaultDependencies=no`?

**Comment by fsateler** [comment]
you could set that via a dropin: `systemctl edit usr.mount`

I wonder if the fstab generator should do this by default

**Comment by niksajur** [comment]
@fsateler No, it doesn't work. Both error messages still appears during reboot and are logged in journal. So, the problem is not gone. But even if it were, I would consider it killing the problem, not solving it.

**Comment by poettering** [comment]
> Maybe /usr and /var should get DefaultDependencies=no , and have systemd-shutdown unmount them (which is already happening).

/usr is already considered "extrensic" in current git, and won't get conflicts dependencies on umount.target. However, /var still will. But maybe we shouldn't. Dunno.

**Comment by poettering** [comment]
@niksajur what are the precise dependencies usr.mount and var.mount have on your system? i.e. do "systemctl show /var /usr" to see them.

**Comment by poettering** [comment]
The /var thing is a duplicate of #867 btw. I am mostly interested in the question why /usr is being stopped at shutdown here...

**Comment by zifxify** [comment]
I have the same error.
$ systemctl show /var /usr

```
Id=var.mount
Names=var.mount
Requires=system.slice systemd-fsck@dev-mapper-vg0\x5cx2dvar.service -.mount
BindsTo=dev-mapper-vg0\x5cx2dvar.device
RequiredBy=systemd-rfkill.service local-fs.target basic.target dbus.socket systemd-backlight@leds:dell::kbd_backlight.service virtlogd.socket systemd-timesyncd.service virtlockd.socket apt-daily.timer systemd-up
WantedBy=dev-mapper-vg0\x5cx2dvar.device
Conflicts=umount.target
Before=roothints.timer virtlockd.socket dbus.socket systemd-backlight@backlight:intel_backlight.service systemd-update-utmp-runlevel.service systemd-journal-flush.service fstrim.timer systemd-update-utmp.service
After=system.slice systemd-fsck@dev-mapper-vg0\x5cx2dvar.service dev-mapper-vg0\x5cx2dvar.device local-fs-pre.target -.mount
RequiresMountsFor=/ /dev/mapper/vg0-var
```

(usr not on a separate partition)

```
Id=usr.mount
Names=usr.mount
Description=usr.mount
LoadState=not-found
ActiveState=inactive
SubState=dead
```

**Comment by poettering** [comment]
@zifxify again, the /var thing is a duplicate of #867 and not interesting as a separate bug report.

And are you sure you can a log message about /usr? Your usr.mount unit data you provided suggests that the unit doesn't exist at all on your system.

Anyway, @niksajur could you provide the relevant data about /usr from your system?

**Comment by ZjYwMj** [comment]
Not @niksajur. I do have /var and /usr on a separates LV. Do I have the same issue?
```
# journalctl -o cat -r -b -1 --unit=mkinitcpio-generate-shutdown-ramfs.service
Started Generate shutdown-ramfs.
==> Build complete.
Starting Generate shutdown-ramfs...
  -> Running build hook: [sd-shutdown]
==> Starting build: none
```
```
# journalctl -o cat -r -b -1 –unit=var.mount
Failed unmounting /var.
var.mount: Mount process exited, code=exited status=32
Unmounting /var...
Mounted /var.
Mounting /var...
```
```
# journalctl -o cat -r -b -1 –unit=usr.mount
Failed unmounting /usr.
usr.mount: Mount process exited, code=exited status=32
Unmounting /usr...
```
```
Systemctl show /var | grep 'Id\|Names\|Require\|BindsTo\|Wanted\|Conflicts\|Before\|After'
Id=var.mount
Names=var.mount
Requires=systemd-fsck@dev-disk-by\x5cx2duuid-b04c0425\x5cx2db1c8\x5cx2d45bd\x5cx2d92ac\x5cx2dc1c49f3e1816.service -.mount system.slice
BindsTo=dev-disk-by\x5cx2duuid-b04c0425\x5cx2db1c8\x5cx2d45bd\x5cx2d92ac\x5cx2dc1c49f3e1816.device
RequiredBy=systemd-update-utmp.service systemd-backlight@backlight:intel_backlight.service systemd-rfkill.service logrotate.timer man-db.service shadow.timer systemd-random-seed.service var-lib-machines.mount basic.target man-db.timer local-fs.target systemd-coredump@0.service systemd-journal-flush.service
WantedBy=dev-disk-by\x5cx2duuid-b04c0425\x5cx2db1c8\x5cx2d45bd\x5cx2d92ac\x5cx2dc1c49f3e1816.device
Conflicts=umount.target
Before=umount.target logrotate.timer local-fs.target systemd-update-utmp.service basic.target var-lib-machines.mount systemd-journal-flush.service man-db.timer man-db.service systemd-backlight@backlight:intel_backlight.service systemd-coredump@0.service shadow.timer systemd-rfkill.service systemd-random-seed.service
After=system.slice local-fs-pre.target systemd-fsck@dev-disk-by\x5cx2duuid-b04c0425\x5cx2db1c8\x5cx2d45bd\x5cx2d92ac\x5cx2dc1c49f3e1816.service -.mount dev-disk-by\x5cx2duuid-b04c0425\x5cx2db1c8\x5cx2d45bd\x5cx2d92ac\x5cx2dc1c49f3e1816.device
RequiresMountsFor=/ /dev/disk/by-uuid/b04c0425-b1c8-45bd-92ac-c1c49f3e1816
```
```
Systemctl show /usr | grep 'Id\|Names\|Require\|BindsTo\|Wanted\|Conflicts\|Before\|After'
Id=usr.mount
Names=usr.mount
Requires=systemd-fsck@dev-disk-by\x5cx2duuid-eb82ba0d\x5cx2d3ccb\x5cx2d4710\x5cx2db02c\x5cx2dc75c89d3cdc2.service -.mount system.slice
BindsTo=dev-mapper-vg0\x5cx2dvg0\x5cx2d\x5cx2dusr.device
RequiredBy=local-fs.target
WantedBy=dev-mapper-vg0\x5cx2dvg0\x5cx2d\x5cx2dusr.device
Before=local-fs.target
After=systemd-fsck@dev-disk-by\x5cx2duuid-eb82ba0d\x5cx2d3ccb\x5cx2d4710\x5cx2db02c\x5cx2dc75c89d3cdc2.service dev-mapper-vg0\x5cx2dvg0\x5cx2d\x5cx2dusr.device -.mount system.slice
RequiresMountsFor=/ /dev/disk/by-uuid/eb82ba0d-3ccb-4710-b02c-c75c89d3cdc2
```

**Comment by poettering** [comment]
Fix waiting in #13010

**Comment by rkdevi27** [comment]
Hi,

I face the unmount of /var/log issue during reboot since journald is using /var/log/journal.
I'm running systemd 241 and don't want to update.

I have written a new service like below and place it before the /var/log/unmount.
 -----------------------------------------------------
/etc/systemd/system/new.service
[Unit]
Description=NEW SERVICE
After=var-log.mount syslog.socket 

[Service]
Type=oneshot
ExecStop=/usr/bin/host_unmount.sh journal_stop
TimeoutStopSec=120
RemainAfterExit=yes
Environment=SYSTEMD_LOG_LEVEL=debug


[Install]
WantedBy=multi-user.target
------------------------------------------------
/usr/bin/host_unmount.sh - this will do stop the journal services as below.

    systemctl stop systemd-journald.service
    systemctl stop systemd-journald.socket
    systemctl stop systemd-journald-audit.socket
    systemctl stop systemd-journald-dev-log.socket

During reboot I see the new.service stop gets timeout even if i get timeout value as 300secs and SIGTERM terminates the process.

The debug shows:

Oct 24 10:40:29 systemd[1]: Started NEW SERVICE.
Oct 24 10:49:58 systemd[1]: Stopping NEW SERVICE...
Oct 24 10:49:58 host_umount.sh[4718]: Bus n/a: changing state UNSET → OPENING
Oct 24 10:49:58 host_umount.sh[4718]: Bus n/a: changing state OPENING → AUTHENTICATING
Oct 24 10:49:58 host_umount.sh[4718]: Executing dbus call org.freedesktop.systemd1.Manager StopUnit(systemd-journald.service, replace)
Oct 24 10:49:58 host_umount.sh[4718]: Bus n/a: changing state AUTHENTICATING → RUNNING
Oct 24 10:49:58 host_umount.sh[4718]: Sent message type=method_call sender=n/a destination=org.freedesktop.systemd1 path=/org/freedesktop/systemd1 interface=org.freedesktop.systemd1.Manager member=StopUnit cookie=1 reply_cookie=0 signature=ss error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=method_return sender=org.freedesktop.systemd1 destination=n/a path=n/a interface=n/a member=n/a cookie=4 reply_cookie=1 signature=o error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Sent message type=method_call sender=n/a destination=org.freedesktop.systemd1 path=/org/freedesktop/systemd1 interface=org.freedesktop.systemd1.Manager member=GetUnit cookie=2 reply_cookie=0 signature=s error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=method_return sender=org.freedesktop.systemd1 destination=n/a path=n/a interface=n/a member=n/a cookie=8 reply_cookie=2 signature=o error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Sent message type=method_call sender=n/a destination=org.freedesktop.systemd1 path=/org/freedesktop/systemd1/unit/systemd_2djournald_2eservice interface=org.freedesktop.DBus.Properties member=Get cookie=3 reply_cookie=0 signature=ss error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=method_return sender=org.freedesktop.systemd1 destination=n/a path=n/a interface=n/a member=n/a cookie=9 reply_cookie=3 signature=v error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Adding /org/freedesktop/systemd1/job/769 to the set
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=signal sender=org.freedesktop.systemd1 destination=n/a path=/org/freedesktop/systemd1/unit/systemd_2djournald_2eservice interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=1 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=signal sender=org.freedesktop.systemd1 destination=n/a path=/org/freedesktop/systemd1/unit/systemd_2djournald_2eservice interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=2 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=signal sender=org.freedesktop.systemd1 destination=n/a path=/org/freedesktop/systemd1 interface=org.freedesktop.systemd1.Manager member=JobNew cookie=3 reply_cookie=0 signature=uos error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=signal sender=org.freedesktop.systemd1 destination=n/a path=/org/freedesktop/systemd1/unit/systemd_2djournal_2dflush_2eservice interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=5 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=signal sender=org.freedesktop.systemd1 destination=n/a path=/org/freedesktop/systemd1/unit/systemd_2djournal_2dflush_2eservice interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=6 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=signal sender=org.freedesktop.systemd1 destination=n/a path=/org/freedesktop/systemd1 interface=org.freedesktop.systemd1.Manager member=JobNew cookie=7 reply_cookie=0 signature=uos error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=signal sender=org.freedesktop.systemd1 destination=n/a path=/org/freedesktop/systemd1/unit/networking_2eservice interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=10 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=signal sender=org.freedesktop.systemd1 destination=n/a path=/org/freedesktop/systemd1/unit/networking_2eservice interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=11 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=signal sender=org.freedesktop.systemd1 destination=n/a path=/org/freedesktop/systemd1/unit/systemd_2duser_2dsessions_2eservice interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=12 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=signal sender=org.freedesktop.systemd1 destination=n/a path=/org/freedesktop/systemd1/unit/systemd_2duser_2dsessions_2eservice interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=13 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=signal sender=org.freedesktop.systemd1 destination=n/a path=/org/freedesktop/systemd1 interface=org.freedesktop.systemd1.Manager member=JobRemoved cookie=14 reply_cookie=0 signature=uoss error-name=n/a error-message=n/a
Oct 24 10:49:58 host_umount.sh[4718]: Got message type=signal sender=org.freedesktop.systemd1 destination=n/a path=/org/freedesktop/systemd1/unit/systemd_2duser_2dsessions_2eservice interface=org.freedesktop.DBus.Properties member=Prop



Can you please explain why there is a timeout? and let me know if there is a way to get it done without getting timed out?
If I stop journal services before initiating the reboot, it works fine.
